app/models/user.py

```diff
@@ -0,0 +1,20 @@
+from sqlmodel import SQLModel, Field
```

Comment:
This needs either commenting or documentation to explain the general principles of how these model classes work